packages/dev/docs/pages/react-aria/ssr.mdx

```diff
@@ -48,6 +50,204 @@ function App() {
 
 See the [internationalization docs](internationalization.html) for more information about i18n in React Aria.
 
```

Comment:
Is it confusing that the above section says the app should be wrapped in an I18nProvider, except you don't need that if you are using LocalizedStringProvider as an optimization as described below? How should we describe that? Should we only describe the optimized version on this page and get rid of the above content?

Reply:
I think a note mentioning this caveat near the beginning of the "Optimizing bundle size" would be helpful section.